packages/sd-jwt/tests/setup.ts

```diff
@@ -0,0 +1,3 @@
+import 'reflect-metadata'
+
+jest.setTimeout(120000)
```

Comment:
Do you expect SD-JWT to take that long?

Reply:
Copied from another test. It is quite quick, but askar and agent initialization makes some steps a bit slow. Better overestimate here I guess ;).